src/patternfly/demos/Banner/__banner-content.hbs

```diff
@@ -0,0 +1,14 @@
+{{#> l-flex l-flex--modifier="pf-m-justify-content-center pf-m-justify-content-space-between-on-lg pf-m-nowrap"}}
```

Comment:
I think you can drop `-content` in the partial name or use `-demo` instead to distinguish from potential component content.